app/gui/view/src/code_editor.rs

```diff
@@ -100,8 +100,9 @@ impl View {
             frp.source.is_visible <+ bool(&frp.input.hide,&frp.input.show);
             frp.source.is_visible <+ frp.toggle.map2(&is_visible, |(),b| !b);
 
+            def init = source_();
             let shape  = app.display.scene().shape();
-            position <- all_with(&height_fraction.value,shape, |height_f,scene_size| {
+            position <- all_with3(&height_fraction.value,shape,&init, |height_f,scene_size,_init| {
```

Comment:
Check youtrack for this issue

Reply:
https://enso.youtrack.cloud/issue/IDE-3022

Reply:
This is not an issue. This is by design. New FRP implementation might do it automatically though.